docs/user/examples.rst

```diff
@@ -16,16 +16,16 @@ Sphinx and MkDocs examples
 
    * - Topic
      - Framework
-     - Description
      - Links
-   * - Sphinx
+     - Description
+   * - Basic Sphinx
      - Sphinx
-     - Basic example of using Sphinx
      - `[Git] <https://github.com/readthedocs-examples/example-sphinx-basic/>`__ `[Rendered] <https://example-sphinx-basic.readthedocs.io/en/latest/>`__
-   * - MkDocs
+     - Basic example of using Sphinx
```

Comment:
These could have a bit more description. What is showcased in this verison? `basic example` doesn't tell me much.  

```suggestion
     - Sphinx example with versioning and Python doc autogeneration
```

If it doesn't work well w/ the tables, could be a good reason to break them out into sections.

Reply:
But don't you think that it's much more inviting and easy to read in a table?